### PR TITLE
[docs-only] 1.4.0 Release Notes

### PR DESCRIPTION
**Pull request by pmaier1**
Please review!

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
LGTM

File: docs/ocis/release_notes.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ geekdocEditPath: edit/master/docs/ocis
...
+
+Version 1.4.0 brings new features, bug fixes and further improvements. The accessibility of ownCloud Web has greatly improved, paving the way for WCAG 2.1 compliance. The Infinite Scale platform has received major improvements regarding memory consumption. The user storage quota feature has been implemented and folder sizes are now properly calculated. It is now possible to write log messages to log files and to specify configuration values using a config file.
+
+The most prominent changes in version 1.4.0 comprise:
+
+- ownCloud Web has received many further improvements regarding accessibility and WCAG 2.1 compliance
+- Reduced the memory consumption of the runtime drastically (by a factor of 24) [#1762](https://github.com/owncloud/ocis/pull/1762)
+- Initial quota support to impose storage space restrictions for users (query / set) [#1405](https://github.com/cs3org/reva/pull/1405)
+- Folder sizes are now calculated correctly (tree size accounting) [#1405](https://github.com/cs3org/reva/pull/1405)
+- The media viewer now also works with upper case file extensions (e.g., JPG) [#4627](https://github.com/owncloud/web/pull/4627)
```
Comment (kulmann) [inline docs/ocis/release_notes.md:20]: That's not included in the ocis release!

**Review by kulmann — CHANGES_REQUESTED** [review]
Web has been updated to 2.1.0 in oCIS. There are a lot of changes in master of web, which still need some attention to detail before we release them to oCIS as well. The media viewer change is not from these release notes here is not in web 2.1.0.

File: docs/ocis/release_notes.md (hunk 1 above)
Comment (fschade) [reply docs/ocis/release_notes.md]: fixed

**Review by kulmann — APPROVED** [review]

**Pull request merged**